cfy_manager/components/cli/cli.py

```diff
@@ -53,9 +54,12 @@ def _configure():
 
     use_cmd = ['cfy', 'profiles', 'use', 'localhost',
                '--skip-credentials-validation']
+
+    # TODO: Don't skip validation after CFY-7689 is done
     set_cmd = [
         'cfy', 'profiles', 'set', '-u', username,
-        '-p', password, '-t', 'default_tenant'
+        '-p', password, '-t', 'default_tenant',
+        '-c', INTERNAL_CERT_PATH, '--skip-credentials-validation'
```

Comment:
should be external, no?

Reply:
Well, we're setting the CLI profiles with `localhost`, so I guess it doesn't matter?

Reply:
then it should be CA, not internal, but i'm not sure why is it green